CHANGELOG.txt

```diff
@@ -1,6 +1,9 @@
 # pltraining/classroom
 ## Release Notes
 
+## v1.5.32
```

Comment:
don't include a version in the changelog on incremental changes. It makes it too hard to keep track of what's released or not. Either don't add anything until release (and get that info from git log when it's time) or add a bullet under an "unreleased" heading or the like.